server/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@markdoc/language-server",
+  "name": "@fjorn/markdoc-ls",
```

Comment:
I think this change probably wasn't intended here, right?

Reply:
oops left that in while installing under a separate name to test